pyproject.toml

```diff
@@ -15,7 +15,7 @@ description = "Ophyd devices and other utils that could be used across DLS beaml
 dependencies = [
     "click",
     "ophyd",
-    "ophyd-async>=0.8.0a1",
+    "ophyd-async @ git+https://github.com/bluesky/ophyd-async.git@adr-device",
```

Comment:
Should: This branch doesn't exist any more, we should pin to main I guess. For testing this at least